### DISCUSSION TITLE
multi-tenancy support in future

### DISCUSSION THREAD

**Discussion by Taraskobilskiy** [Security-Onion-Solutions/securityonion#13311] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

32

### RAM

256

### Storage for /

4T

### Storage for /nsm

2T

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Hello SecurityOnion Community,

We are actively using Security Onion and greatly appreciate its functionality. As our business processes expand, we have identified a need for multi-tenancy support.

We have noticed that the platform currently does not support this functionality. In this regard, we would like to inquire:

Are there any plans to implement multi-tenancy functionality in SecurityOnion in the near future?
If so, are there any estimated timelines for the deployment of this functionality?
If not, is it technically feasible within the current architecture of the platform?
We would be grateful for any information and recommendations you can provide.

Sincerely

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
We are considering options for the future, but we have no timelines or any other information to share at this point.
